<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-CPTC2M6Q" failures="1" tests="1" name="Test" time="15.376" errors="0" timestamp="2023-04-29T07:19:02 IST">
  <testcase classname="test.com.testR8" name="testcase1" time="15.224">
    <failure type="java.lang.AssertionError" message="expected [6] but found [0]">
      <![CDATA[java.lang.AssertionError: expected [6] but found [0]
at test.com.testR8.loginUser(testR8.java:27)
at test.com.testR8.testcase1(testR8.java:49)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testcase1 -->
</testsuite> <!-- Test -->
